tests/test_openspiel.py

```diff
@@ -148,8 +148,8 @@ def test_seeding(game):
     env1.reset(seed=42)
     env2.reset(seed=42)
 
-    agent1 = env1.agent_iter()
-    agent2 = env2.agent_iter()
+    agent1 = env1.agents[0]
+    agent2 = env2.agents[0]
```

Comment:
I don't think this bit is a problem because it's just hard coding the first agent name in order to do the action space seeding, we still use the iter.